src/views/legal/Privacy.vue

```diff
@@ -0,0 +1,136 @@
+<template>
```

Comment:
![Screenshot from 2023-08-09 10-32-39](https://github.com/dumpus-app/dumpus-landing/assets/42497995/b8085a77-ee3d-45c6-b8bb-845021f0cef6)
there is an issue with all the pages, the background does not last until the bottom of the page. Do you know why this is happening?

Reply:
have fixed it